src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/models/rgw-storage-class.model.ts

```diff
@@ -226,3 +245,68 @@ export const RESTORE_STORAGE_CLASS_TEXT = $localize`The storage class to which o
 export const ZONEGROUP_TEXT = $localize`A Zone Group is a logical grouping of one or more zones that share the same data
                   and metadata, allowing for multi-site replication and geographic distribution of
                   data.`;
+
+export type AclType = 'id' | 'email' | 'uri';
+
+export interface AclLabelAndHelper {
+  source: string;
+  destination: string;
+}
+
+export interface AclMaps {
+  [key: string]: AclLabelAndHelper & {
+    [field: string]: string;
+  };
+}
+
+export enum AclLabel {
+  source = 'Source',
+  destination = 'Destination'
+}
+
+export enum AclFieldType {
+  Source = 'source',
+  Destination = 'destination'
+}
+
+export const AclTypeOptions = [
+  { value: 'id', label: 'ID' },
+  { value: 'email', label: 'Email' },
+  { value: 'uri', label: 'URI' }
+] as const;
+
+export const AclTypeConst = {
+  ID: 'id',
```

Comment:
```suggestion
  [AclTypeOptions[0].label]: 'id',
```
reuse , same for others.